content/docs/user-guide/contributing/docs.md

```diff
@@ -81,36 +77,60 @@ Launch the server locally with:
 $ yarn develop
 ```
 
-This will start the server on the default port, `3000`. Visit
-`http://localhost:3000/` and navigate to the docs in question. This will also
-enable the Git pre-commit hook that will be formatting your code and
+This will start the server on the default port, `8000`. Visit
+`http://localhost:8000/` and navigate to the page in question. This will also
+enable the Git pre-commit hook that will be formatting and linting your code and
 documentation files automatically.
 
-### Debugging
+### All available commands
+
+To build the project and run it:
+
+- `yarn build` - build assets in the `public` directory (`gatsby build`).
+- `yarn develop` - run development server with hot reload (`gatsby develop`).
+- `yarn start` - run production static server over the `public` directory.
+
+If you change source code files, run tests:
+
+- `yarn test` - run tests.
+
+We use [`prettier`](https://prettier.io/) to format our source code, below is a
+set of pre-built commands for your convenience. All of these checks will be
+enforced automatically upon [submitting PRs](#submitting-changes).
+
+- `yarn format-check` - check all source and content files that they are
+  properly formatted. This command does not fix any found issue, only reports
+  them.
+- `yarn format-all` - fix all found problems.
+- `yarn format-staged` - same, but only on staged files.
+- `yarn format` - run this command `yarn format <file-name>` to format a
+  specific file.
+
+We use `tsc`, `eslint`, and other linters to check source code style and find
+different errors. All of these checks will be enforced automatically upon
+[submitting PRs](#submitting-changes).
+
+- `yarn lint-ts` - lint source code files (`.ts`, `.js`, `tsx`, etc).
+- `yarn lint-css` - lint `.css` files.
+
+Other commands:
+
+- `yarn link-check` - runs script to detect broken URLs (e.g. those that return
+  404 - Not Found) in the content.
 
-The `yarn debug` script runs the local development server with `node`'s
-[`--inspect-brk` option](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options)
-in order for debuggers to connect to it (on the default port, 9229).
+### ENV variables
 
-> For example, use this launch configuration in **Visual Studio Code**:
->
-> ```json
-> {
->   "type": "node",
->   "request": "launch",
->   "name": "Launch via Yarn",
->   "runtimeExecutable": "yarn",
->   "runtimeArgs": ["debug"],
->   "port": 9229
-> }
-> ```
+There are environment variables production code depends on (e.g. to deploy
+itself properly), or those that can be used to debug the project. Please, check
+the production system settings to see all variables that production and
+deployment system depend on.
 
-### Running tests
+Some useful variables:
```

Comment:
So these are just the optional ones?

Reply:
no, I was really hesitant to keep this section at all ... I don't want to support it, but decided to mention this just in case next JS folks see this and will also at least ask question how to get the full list

Reply:
OK. But I guess you don't need any of these for local dev. I don't have any, for example.

Reply:
You need actually, if you want to test certain production-related engine things locally. But again, this section is here purely for JS devs, to give them a hint about prod vars, etc